glide.lock

```diff
@@ -263,27 +236,75 @@ imports:
   version: 3887ee99ecf07df5b447e9b00d9c0b2adaa9f3e4
 - name: gopkg.in/yaml.v2
   version: 53feefa2559fb8dfa8d81baad31be332c97d6c77
+- name: k8s.io/apimachinery
```

Comment:
Yeah.. All API and models are being done here. Kubernetes did this migration not too long ago.